Cabal/src/Distribution/Simple/Setup/Config.hs

```diff
@@ -319,7 +323,9 @@ instance Eq ConfigFlags where
       && equal configDebugInfo
       && equal configDumpBuildInfo
       && equal configUseResponseFiles
+      && equal configAllowDependingOnPrivateLibs
```

Comment:
Is this a leak from the private dependencies PR?

Reply:
No, this was missing from the `Eq` check. It's about private sub-libraries (as in, not visible)

Reply:
Good catch!